index.html

```diff
@@ -4588,6 +4410,7 @@ <h5>Note regarding the ARIA 1.3 <code>image</code> role.</h5>
 							<ul>
 								<li><code>&lt;ol&gt;</code> in [[HTML]]</li>
 								<li><code>&lt;ul&gt;</code> in [[HTML]]</li>
```

Comment:
worth adding in `<menu>` here as part of this pr?  or want me to create a separate PR just for this?

Reply:
I have never seen menu used anywhere so not sure I see the benefit of adding it.

Reply:
merely to be complete, since the spec lists 3 of the 4 related elements.

Reply:
I don't think it makes the spec any more understandable though as most devs would probably mistake the menu element for something useful